test/matrix/csr_kernels2.cpp

```diff
@@ -240,9 +275,35 @@ TEST_F(Csr, SimpleApplyIsEquivalentToRefWithOnemkl)
 }
 
 
-TEST_F(Csr, SimpleApplyIsEquivalentToRefWithOnemklUnsorted)
```

Comment:
Isnt this this specfic to DPCPP ?

Reply:
We test the behavior in SimpleApplyIsEquivalentToRefWithSparselibUnsorted